packages/charts/src/utils/tooltip.utils.ts

```diff
@@ -54,7 +58,7 @@ export const getCircularChartTooltipContent = ({
     const total = keys.reduce((acc, keyName) => {
       return acc + item[keyName];
     }, 0);
-    const formattedTotal = formatValue ? formatValue(total) : total;
+    const formattedTotal = valueFormatter(total, formatValue);
```

Comment:
Please, add unit tests for this `TooltipFormatter` usage. Additionally I believe we could reducer technical debt and increase test coverage for this function.